pkg/alertmanager/multitenant_test.go

```diff
@@ -2520,10 +2520,20 @@ func TestComputeConfig(t *testing.T) {
 	cfg2.GrafanaAlertmanagerTenantSuffix = "-grafana"
 	amWithSuffix := setupSingleMultitenantAlertmanager(t, cfg2, store, nil, featurecontrol.NoopFlags{}, log.NewNopLogger(), reg2)
 
-	tenantReceivingRequests := "receiving-1-grafana"
-	amWithSuffix.lastRequestTime.Store(tenantReceivingRequests, time.Now().Unix())
-	tenantReceivingRequestsExpired := "receiving-2-grafana"
-	amWithSuffix.lastRequestTime.Store(tenantReceivingRequestsExpired, time.Now().Add(-time.Hour).Unix())
+	reg3 := prometheus.NewPedanticRegistry()
+	cfg3 := mockAlertmanagerConfig(t)
+	cfg3.StrictInitialization = true
+	amWithStrictInit := setupSingleMultitenantAlertmanager(t, cfg3, store, nil, featurecontrol.NoopFlags{}, log.NewNopLogger(), reg3)
```

Comment:
I needed a third config + registry + AM to test:
- Alertmanager w/o strict initialization
- Alertmanager w/ strict initialization just for Grafana tenants
- Alertmanager w/ strict initialization for all tenants